apps/web/src/components/dashboard/energy-consumption-tooltip.tsx

```diff
@@ -54,14 +54,25 @@ export default function EnergyConsumptionTooltip({ payload }: TooltipProps<Value
         }
     };
 
+    const computeTimestampLabel = (aggregationParam) => {
+        switch (aggregationParam) {
+            case AggregationType.YEAR: return 'Wh / Jahr';
+            case AggregationType.MONTH: return 'Wh / Monat';
+            case AggregationType.WEEK: return 'Wh / Wochen';
+            case AggregationType.DAY: return 'Wh / Tag';
+            case AggregationType.HOUR: return 'Wh / Stunde';
+            default: return 'Wh (Einheit nicht spezifiziert)';
+        }
+    };
+
```

Comment:
Können wir diese Funktion generell verfügbar machen in einem package wie lib. Dann können wir diese auch im Admin Dashboard wieder integrieren. Auch können wir die Funktion so gestallten, dass es sich nicht doppelt.

Reply:
Die Funktion ins lib package zu verschieben, könnte ich auch im Zuge von #47 machen. Die beiden PRs sind ohnehin kon­flik­tär und ich kann die Konflikte wahrscheinlich einfacher lösen. Dann würde ich mit meinem PR so lange warten, bis der PR hier durch ist.

`Auch können wir die Funktion so gestallten, dass es sich nicht doppelt.`
@RitaTagoula Kannst du diesen Punkt dann noch bitte einarbeiten und dann mergen, wenn nicht noch weitere Anmerkungen kommen?